plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/handler/EditResourceHandler.java

```diff
@@ -67,25 +66,18 @@ private void openInEditor(IWorkbenchWindow window, Connection connection, IResou
 		   if (page != null) {
 			   IEditorRegistry editorRegistry= PlatformUI.getWorkbench().getEditorRegistry();
 			   String defaultJsonEditorId = getDefaultJSONEditorDescriptorId(editorRegistry);
-			   IEditorDescriptor editorDescriptor = editorRegistry.getDefaultEditor(resource.getName()+".json", null);
-			   String editorId = editorDescriptor == null?defaultJsonEditorId:editorDescriptor.getId();
-			   try {
-				   page.openEditor(input, editorId);
-			   } catch (PartInitException O_o) {
-				   if (defaultJsonEditorId.equals(editorId)){
-					   throw O_o;
-				   }
-				   //try to fall back on default editor
-				   page.openEditor(input, defaultJsonEditorId);
-			   }
+			   page.openEditor(input, defaultJsonEditorId);
```

Comment:
Yes but we would be facing the switch to the JSON editor depending on the user Eclipse configuration